tests/typescript-mock-data.spec.ts

```diff
@@ -168,3 +168,12 @@ it('should generate mock data with as-is types and enums if typenames is "keep"'
     expect(result).not.toMatch(/ABC(TYPE|STATUS)/);
     expect(result).toMatchSnapshot();
 });
+
+it('should generate the `casual` data for a particular scalar mapping', async () => {
+    const result = await plugin(testSchema, [], { scalars: { AnyObject: 'email' } });
+
+    const emailRegex = /(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))/;
+    expect(result).toBeDefined();
+    expect(emailRegex.test(result as string)).toBeTruthy();
+    expect(result).toMatchSnapshot();
+});
```

Comment:
Could you add a test for custom scalar that is not a string?

Reply:
sure thing. Will update soon and ping

Reply:
Thanks for the second test, but it seems like both test generate a string
https://github.com/ardeois/graphql-codegen-typescript-mock-data/pull/21/files/a3511d4a629bedb2a231dc74197ee166e315ad1c..b43a92e5b172f8d035419270cbc65acf80dd12cb#diff-ce1f2f758ad7b0b108a11d9a0171e13dR34

Shouldn't you test with a number?

Reply:
Yeah updated it to something a bit more extreme just to be safe